README.md

```diff
@@ -1,8 +1,8 @@
 # goreadme
 
-[![Build Status](https://travis-ci.org/posener/goreadme.svg?branch=master)](https://travis-ci.org/posener/goreadme)
```

Comment:
revert changes in this file.

Reply:
Aha thanks, that was not intentional!
The side effects of having GitHub Actions automatically updating READMEs! :sweat_smile: